lib/Cargo.toml

```diff
@@ -58,24 +58,25 @@ async-recursion = "1.0.4"
 base64_lib = { version = "0.21.2", package = "base64" }
 bcrypt = "0.14.0"
 bincode = "1.3.3"
-bung = "0.1.0"
+bytes = "1.4.0"
+cedar-policy = "2.3.2"
 channel = { version = "1.9.0", package = "async-channel" }
 chrono = { version = "0.4.26", features = ["serde"] }
-derive = { version = "0.9.0", package = "surrealdb-derive" }
+derive = { version = "0.11.0", package = "surrealdb-derive" }
```

Comment:
Happy to keep this in because it speeds things up. I would like package upgrades to be separate PRs though. This is because we can track failures to package upgrades then.

It's marginally easier than tracking to a surface change that also upgrades packages, so disputable usefulness. But it does help in quickly eliminating changes that break builds.

Reply:
A good point with regards to the other packages. `SurrealDB-derive` is heavily integral to this PR though and needs to be upgraded in this PR. Separately and it will fail!